### DISCUSSION TITLE
This page is taking too long to load.  Sorry about that. Please try refreshing and contact us if the problem persists.

### DISCUSSION THREAD

**Discussion by Sapkotaanish** [community/community#35026] [Other Feature Feedback, Questions, & Ideas] [answered]
Whenever I try to connect my GitHub account to any third parties, I get the Unicorn page.
I get same error while trying to connect my GitHub account to DigitalOcean, CloudFlare and Vercel.
It has been 3-4 days since I am facing this issue.
How can it be fixed?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Sapkotaanish! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

**Comment by 3kh0** [comment]
This happens every so often, here are a few quick fixes that sometimes help.

- Reload the page (easiest in the book)
- Reload and clear cache, in chrome do ctrl+shift+r
- Disconnect from wifi, and reconnect
- Open cmd and do `ipconfig/release` and then `ipconfig/renew`
- Turn off your computer, wait 10 seconds, and turn it back on.
- Try a different browser
- Clear cookies and site data from github.com
- Reboot your router

These are some quick and easy fixes, this issue happens from time to time again, and these fixes seem to fix it for me.

*If this helped please mark this as the answer!* ❤️

**Reply by thematheusmello** [reply]
> This happens every so often, here are a few quick fixes that sometimes help.
> 
> * Reload the page (easiest in the book)
> * Reload and clear cache, in chrome do ctrl+shift+r
> * Disconnect from wifi, and reconnect
> * Open cmd and do `ipconfig/release` and then `ipconfig/renew`
> * Turn off your computer, wait 10 seconds, and turn it back on.
> * Try a different browser
> * Clear cookies and site data from github.com
> * Reboot your router
> 
> These are some quick and easy fixes, this issue happens from time to time again, and these fixes seem to fix it for me.
> 
> _If this helped please mark this as the answer!_ ❤️

I am still facing this problem after doing this.
I also tried to connect using different Wifis but it didn't work.
I believe this is a problem from the GitHub side, not my machine.
The weird thing is that I faced this only when I was creating a new project on Vercel but when I tried logging in using the GitHub option it worked.

![image](https://user-images.githubusercontent.com/15657692/195068568-d165eb0e-4517-4d50-9ab0-ce3e18f268ab.png)

**Reply by 0xAskar** [reply]
I'm having the exact same problem it only happens when I try to create a new project. Any help? Also, wdym @matsmello by logging in using the GitHub option? Cause that's when I get the error

**Comment by jfullstackdev** [comment]
installed extensions on the browser can really affect GitHub, that's quite common occurrence, 

try logging in in Incognito mode

or it's really on the end of those external services, better contact them

**Comment by imanishbarnwal** [comment]
Hi, I am facing the same issue I can't connect my GitHub organization with netlify whenever I'm trying to connect it shows me something like this. Can anyone please help me with this?

![image](https://user-images.githubusercontent.com/46371923/194758123-ee749f9b-5650-4cc9-861b-f8a6ca8fc5f4.png)

**Reply by 3kh0** [reply]
Try https://github.com/orgs/community/discussions/35026?notification_referrer_id=NT_kwDOA3Z_zLM0NTMwOTkwMjQ5OjU4MDk3NjEy#discussioncomment-3818894

**Reply by imanishbarnwal** [reply]
Tried every sort of thing but didn't work for me!! :'(

**Reply by Sapkotaanish** [reply]
Yup, same for me

**Reply by imanishbarnwal** [reply]
Did you contact GitHub? And got any response from them regarding this? @Sapkotaanish

**Reply by Sapkotaanish** [reply]
No, I don't know where to contact them.

**Reply by imanishbarnwal** [reply]
In the screenshot I shared there's an option to contact support you should use that. I did the same. Now waiting for their response.

**Reply by Sapkotaanish** [reply]
Ah Okay, Please let me know if you get any solution from them.

**Reply by 0xAskar** [reply]
Could anyone here update me if they find a solution

**Reply by Sapkotaanish** [reply]
@0xAskar 
https://github.com/orgs/community/discussions/35026#discussioncomment-3878922 
this solution by @m2mt-l fixed the issue for me.

**Comment by Sapkotaanish** [comment]
I also tried every every mentioned ways, but it didn't work.
I get same error message every time I try to connect anything to GitHub (Vercel, DigitalOcean, Cloudflare).
So I think the error is really on GitHub side.

**Comment by cheesycod** [comment]
same issue here

**Comment by 0xAskar** [comment]
I have the same exact issue on both Vercel and Netlify. Definitely a Github problem.

**Comment by m2mt-l** [comment]
I have also the same issue with Vercel and Netlify.

**Comment by m2mt-l** [comment, marked as answer]
This works for me. After removing one of the organizations as collaborator the issue has been resolved.
1. Go to: Settings -> Organizations
2. Remove yourself as collaborator (if possible) for organizations with many repositories
https://github.com/orgs/community/discussions/34751

**Reply by imanishbarnwal** [reply]
But how to find or know which organization creating this issue?

**Reply by Sapkotaanish** [reply]
This worked for me. Thanks @m2mt-l. But GitHub should fix it soon.

**Reply by m2mt-l** [reply]
Maybe this should be that you are the collaborator and the organization has many repositories. In my case me50 is the one used by HarvardX's CS50 course as others reported above thread. Agreed GitHub should fix it soon.

**Reply by Sapkotaanish** [reply]
Yup, exact same for me. I also left me50 and it worked.

**Reply by 0xAskar** [reply]
@m2mt-l @Sapkotaanish This worked for me! Thank you very much! For more context, I am a collaborator in two organizations. I **only had to remove myself from me50**. I stayed in the other and it worked without needing to remove myself from that one.

**Reply by m2mt-l** [reply]
Not sure since I don't use me50 anymore. better to contact GitHub support if you don't want to lose the progress.

**Reply by gildoneto** [reply]
Thank you so much @m2mt-l . Same problem here, and when I leave me50 it worked. Now I can connect to Vercel. Thanks for sharing!!!

**Reply by Blankeos** [reply]
@0xAskar @m2mt-l Can you guys check if you lost progress on me50 when you left? I'm like half-way done through cs50 and I don't wanna lose my progress just to try out Railway.

**Reply by Golyn** [reply]
It works now. I removed me50 and other organizations. Thank you very much.

**Reply by gotneb** [reply]
I was facing the same problem. After I've removed me50 it worked.

**Reply by Anthoni-Mathias** [reply]
is work for me is well thank you :)

**Reply by Balavenkatarao-paycraft** [reply]
> It works now. I removed me50 and other organizations. Thank you very much.

HI There, I'm currently facing the same issue, Can you explain me in detail, Where and how you removed the me50.
Can you please help me out, Thank you.

**Reply by Pcniado** [reply]
Doesnt work for me sadly with Unreal Engine

**Reply by donguklim** [reply]
I am also having this trouble with Unreal Engine since last week

**Reply by yuyueplus** [reply]
I am also having this trouble with Unreal Engine since last week too

**Reply by NavySunKOR** [reply]
It doesn't work with unreal engine repository also.

**Reply by DylanReidRamelli** [reply]
For me as well since last week

**Reply by chuncui0207** [reply]
Unreal Engine not work

**Reply by WaleedLojj** [reply]
I was able to access the UE repository a long time ago without an issue. Two days ago, when I tried to access it again, I got the unicorn page. I unlinked and relinked my Epic account to my GitHub account and again got the unicorn page. I am only a member of the Epic Games organization. I tried to remove myself as a collaborator by following the GitHub documentation ("Settings", "Code, planning, and automation", "Repositories"), but the repository list was empty for me.

**Reply by WaleedLojj** [reply]
This is the reply from Epic Games: 

Hello!
 
I would totally love to help you with that issue, but our Unreal Engine part has a separate support. In this instance, I am not sure if this is not a Github issue on its own website, but I believe you might know better how that works. 
Please check this page for help for Unreal Engine:
https://www.unrealengine.com/support
 
 If you have a problem with something about our games or store in the future, don't be afraid to write to us. We will solve your problem as soon as possible.

Best regards
Epic Games Support Agent

**Reply by Pcniado** [reply]
As for more info about my issue, i can't access the nvRTX fork of UE, however i can access the normal one

**Reply by soyobat** [reply]
but i only have two Organizations.
and i try again fork but the problem still exist
thanks

**Comment by medAmineRg** [comment]
hey i just want to fork a project then the unicorn show up.
i dont have any Organizations in my settings.
thanks

**Comment by pe-bijay** [comment]
While payment of Github am getting the unicorn error

**Reply by Balavenkatarao-paycraft** [reply]
HI Bijay, Have you found any Resolution for that ?

**Reply by Balavenkatarao-paycraft** [reply]
Hello Bijay, I Hope you have received a response from the GitHub support team, If problem was resolved, Can you please kindly let me know the resolution process. Thank you

**Comment by pe-bijay** [comment]
No, whenever am trying to do payment its showing below error

[image: image.png]

On Fri, Nov 8, 2024 at 2:21 PM Balavenkatarao Pidatala <
***@***.***> wrote:

> HI Bijay, Have you found any Resolution for that ?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/35026#discussioncomment-11186663>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BIP2PSWGDMPNOZS5CKRHMFLZ7R3PRAVCNFSM6AAAAABRIZD7Z2VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMJYGY3DMMY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Thanks & Regards
Bijay Singh

-- 


Disclaimer:

This email contains proprietary, business-confidential, 
and/or privileged material. The information contained in this e-mail is 
intended solely for the use of the individual to whom it is addressed. If 
you are not the intended recipient, you are hereby notified that any 
disclosure, copying, distribution or action in relation to the contents of 
this email is strictly prohibited and may be unlawful and request you to 
permanently delete this message and any attachments herein. You are 
requested to advise the sender, of your receipt of this email by return 
e-mail. The Company and its affiliates rule out any and every liability 
resulting from this or any other electronic transmission.

**Reply by Balavenkatarao-paycraft** [reply]
Yes, I'm also experiencing the same,I have raised a ticket to the GitHub support team to resolve this ASAP as we are approaching the bill payment due date.

**Comment by pe-bijay** [comment]
I have also raised a ticket yesterday but have not received any response
yet.

Do let me know if you get any response.

On Fri, Nov 8, 2024 at 1:04 AM Balavenkatarao Pidatala <
***@***.***> wrote:

> Yes, I'm also experiencing the same,I have raised a ticket to the GitHub
> support team to resolve this ASAP as we are approaching the bill payment
> due date.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/35026#discussioncomment-11186772>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BIP2PSUXUI77YK6ZAW6OU3DZ7R5AFAVCNFSM6AAAAABRIZD7Z2VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMJYGY3TOMQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Thanks & Regards
Bijay Singh

-- 


Disclaimer:

This email contains proprietary, business-confidential, 
and/or privileged material. The information contained in this e-mail is 
intended solely for the use of the individual to whom it is addressed. If 
you are not the intended recipient, you are hereby notified that any 
disclosure, copying, distribution or action in relation to the contents of 
this email is strictly prohibited and may be unlawful and request you to 
permanently delete this message and any attachments herein. You are 
requested to advise the sender, of your receipt of this email by return 
e-mail. The Company and its affiliates rule out any and every liability 
resulting from this or any other electronic transmission.

**Reply by Balavenkatarao-paycraft** [reply]
Sure, Its been three days that i raised the ticket, No any response received yet. I will inform you as soon as I receive any update from them. In the meantime, could you please let me know if the issue gets resolved? Thank you.

**Comment by pe-bijay** [comment]
Not yet, not received any update from the support team.

On Mon, Nov 11, 2024 at 10:38 AM Balavenkatarao Pidatala <
***@***.***> wrote:

> Hello Bijay, I Hope you have received a response from the GitHub support
> team, If problem was resolved, Can you please kindly let me know the
> resolution process. Thank you
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/35026#discussioncomment-11210532>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BIP2PSU4JVAV5EAK4EMZGEL2AA3TLAVCNFSM6AAAAABRIZD7Z2VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMRRGA2TGMQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Thanks & Regards
Bijay Singh

-- 


Disclaimer:

This email contains proprietary, business-confidential, 
and/or privileged material. The information contained in this e-mail is 
intended solely for the use of the individual to whom it is addressed. If 
you are not the intended recipient, you are hereby notified that any 
disclosure, copying, distribution or action in relation to the contents of 
this email is strictly prohibited and may be unlawful and request you to 
permanently delete this message and any attachments herein. You are 
requested to advise the sender, of your receipt of this email by return 
e-mail. The Company and its affiliates rule out any and every liability 
resulting from this or any other electronic transmission.

**Reply by Balavenkatarao-paycraft** [reply]
Hello Bijay, Have you received any revert from the support team about the resolution ?

**Reply by pe-bijay** [reply]
> Hello Bijay, Have you received any revert from the support team about the resolution ?

Hi Balavenkatarao,
Any update at your end for payment issue

**Reply by Balavenkatarao-paycraft** [reply]
Hello Bijay,

I regret to inform you that I haven’t received any support from the team yet, despite following up with eight emails over the past week. I’m quite disappointed with the lack of response, especially as we have already passed the due date.

Do you have any information on the potential impact of missing this deadline?

**Comment by pe-bijay** [comment]
Not yet

On Tue, Nov 12, 2024 at 2:55 PM Balavenkatarao Pidatala <
***@***.***> wrote:

> Hello Bijay, Have you received any revert from the support team about the
> resolution ?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/35026#discussioncomment-11223474>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BIP2PSSCUOBGWNF6X5GIRXT2AHCPHAVCNFSM6AAAAABRIZD7Z2VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMRSGM2DONA>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Thanks & Regards
Bijay Singh

-- 


Disclaimer:

This email contains proprietary, business-confidential, 
and/or privileged material. The information contained in this e-mail is 
intended solely for the use of the individual to whom it is addressed. If 
you are not the intended recipient, you are hereby notified that any 
disclosure, copying, distribution or action in relation to the contents of 
this email is strictly prohibited and may be unlawful and request you to 
permanently delete this message and any attachments herein. You are 
requested to advise the sender, of your receipt of this email by return 
e-mail. The Company and its affiliates rule out any and every liability 
resulting from this or any other electronic transmission.

**Comment by pe-bijay** [comment]
today i have received an update from the github support team that it's a
known bug and their team is working on it.

On Thu, Nov 14, 2024 at 12:03 PM Balavenkatarao Pidatala <
***@***.***> wrote:

> Hello Bijay,
>
> I regret to inform you that I haven’t received any support from the team
> yet, despite following up with eight emails over the past week. I’m quite
> disappointed with the lack of response, especially as we have already
> passed the due date.
>
> Do you have any information on the potential impact of missing this
> deadline?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/35026#discussioncomment-11250134>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BIP2PSWTKW7JD4B2J56YES32AQ72TAVCNFSM6AAAAABRIZD7Z2VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMRVGAYTGNA>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Thanks & Regards
Bijay Singh

-- 


Disclaimer:

This email contains proprietary, business-confidential, 
and/or privileged material. The information contained in this e-mail is 
intended solely for the use of the individual to whom it is addressed. If 
you are not the intended recipient, you are hereby notified that any 
disclosure, copying, distribution or action in relation to the contents of 
this email is strictly prohibited and may be unlawful and request you to 
permanently delete this message and any attachments herein. You are 
requested to advise the sender, of your receipt of this email by return 
e-mail. The Company and its affiliates rule out any and every liability 
resulting from this or any other electronic transmission.

**Reply by Balavenkatarao-paycraft** [reply]
Wow, Good to hear 
Have they given you any timeline for the resolution !
